cmd/vocli/commands/keys.go

```diff
@@ -0,0 +1,267 @@
+		}
+
+		// check if directory exists first
+		if _, err := os.Stat(keysDir); os.IsNotExist(err) {
+			return err
+		}
+
+		return filepath.WalkDir(keysDir, func(path string, info os.DirEntry, err error) error {
+			if !info.IsDir() && (filepath.Ext(path) == keyExt) {
+				fmt.Fprintln(Stdout, path)
+			}
+			return nil
+		})
+	},
+}
+
+var keysChangePasswordCmd = &cobra.Command{
+	Use:   "changepassword <filename>",
+	Short: "Changes the password of a keyfile.",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		k, _, err := openKeyfile(args[0], "Enter old password")
+		if err != nil {
+			return err
+		}
+		password = ""
+		passwordNew, err := PromptPassword("Enter new password:")
+		if err != nil {
+			return err
+		}
+		keyJSONNew, err := ethkeystore.EncryptKey(k, passwordNew, scryptN, scryptP)
+		if err != nil {
+			return fmt.Errorf("couldn't encrypt the key with new password: %s", err)
+		}
+
+		return writeKeyFile(args[0], keyJSONNew)
+	},
+}
+
+var keysShowPrivKeyCmd = &cobra.Command{
+	Use:   "showprivkey <filename>",
+	Short: "Prints the hex private key.",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		k, _, err := openKeyfile(args[0], "Enter password:")
+		if err != nil {
+			return err
+		}
+		p := crypto.FromECDSA(k.PrivateKey)
+		pHex := hex.EncodeToString(p)
+		fmt.Fprintf(Stdout, "\n%s\n", pHex)
+		return nil
+	},
+}
+
+func openKeyfile(path, prompt string) (*ethkeystore.Key, *ethereum.SignKeys, error) {
+	keyJSON, err := os.ReadFile(path)
+	if err != nil {
+		return nil, nil, err
+	}
+	password, err := PromptPassword(prompt)
+	if err != nil {
+		return nil, nil, err
+	}
+	k, err := ethkeystore.DecryptKey(keyJSON, string(password))
+	if err != nil {
+		return nil, nil, fmt.Errorf("couldn't decrypt the key with given password: %s", err)
+	}
+
+	signer := ethereum.NewSignKeys()
+	signer.Private = *k.PrivateKey
+	signer.Public = k.PrivateKey.PublicKey
+	return k, signer, nil
+}
+
+// PromptPassword asks the user for a password, but if one was specified through
+// the --password argument, it simply returns that variable without printing
+// anything
+func PromptPassword(prompt string) (string, error) {
+	if password == "" {
+		fmt.Fprint(Stdout, prompt)
+		// if this is a real terminal (not a test case mocked stdin), use the
+		// standard term.ReadPassword() function
+		if term.IsTerminal(int(Stdin.Fd())) {
+			p, err := term.ReadPassword(int(Stdin.Fd()))
+			if err != nil {
+				return "", err
+			}
+			return string(p), nil
+		} else {
+			// if this is a test case and stdin is mocked out, read from Stdin in a
+			// different way
+			reader := bufio.NewReader(Stdin)
+			p, err := reader.ReadString('\n') // returns 'newPassword\n'
+			if err != nil {
+				return "", err
+			}
+			return strings.TrimSpace(p), nil // newPassword
+		}
+	}
+	return password, nil
+}
+
+func createAccount(key *ethkeystore.Key, gatewayRpc, infoUri, faucetPkg string) error {
```

Comment:
Should this function be included in the account.go file as it has to do with accounts ?